### PR TITLE
update version to 0.5.0

### PR DESCRIPTION
**Pull request by hongye-sun**

### CODE REVIEW

**Comment by rmgogogo** [comment]
there is a VERSION file in root folder, it's used for testing and later based on it we use script to update other files

**Comment by rmgogogo** [comment]
https://github.com/kubeflow/pipelines/blob/master/VERSION

**Comment by hongye-sun** [comment]
I updated root version file in the PR. Did you see it?

File: sdk/python/setup.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     # kfp-server-api package is released. Update the lower version when there is
     # a breaking change in kfp-server-api, or kfp sdk depends on new api changes
     # in kfp-server-api.
-    'kfp-server-api>=0.2.5, <0.4.0',
+    'kfp-server-api>=0.2.5, <0.6.0',
```
Comment (rmgogogo) [inline sdk/python/setup.py:33]: good catch, thanks

Comment (Ark-kun) [reply sdk/python/setup.py:33]: `requirements.in` and `requirements.txt` need to be updated as well.

**Pull request merged**